docs/Posting Guide.md

```diff
@@ -1,15 +1,144 @@
-# Will be updated Soon
+# Publishing your blog post
 
+This section deals with the step-by-step process of how to write and publish posts on your site.
 
-A basic Jekyll site usually looks something like this:
+## Make `_pages` directory
+Create a directory named `_pages` in root if you do not have it.
+
+```
+$ mkdir _pages
+$ cd _pages
+```
+
+## Organize your directory structure
+
+Our theme provides a hierarchical directory structure. You may create subdirectories (let's say `Category A`) in `_pages`, and then all post entities placed in the `Category A` directory will be categorized as such.
+
+```
+._pages
+├── Category A
+├── Category B
+|   ├── Subcatecory b
+|   ├── Subcatecory c
+```
+
+### Note: Make sure all directories have an `index.md` inside of them.
+
+The next step is, placing `index.md` files in both `_pages` directory and its subdirectories. The inner content of each `index.md` should be just two dashed lines as shown below:
+
+```
+---
+---
+```
+
+Or you can type the following command from each directory.
+
+```
+$ echo -e "---\n---" > index.md
+```
+
+Your `_pages` structure now looks like this:
+
+```
+._pages
+└── index.md
+├── Category A
+|   └── index.md
+├── Category B
+|   └── index.md
+|   ├── Subcatecory b
+|       └── index.md
+|   ├── Subcatecory c
+|       └── index.md
+```
+
+## Write a blog post
+
+Create a post with file extension: `.md` (ex., *Post-name.md*).  
+
+All blog post files must begin with front matter typically used to set a title or other metadata.
+
+**Note that the `title` and `date` fields can't be left blank.** 
+
+For a simple example:
+
+```
+---
+title: "Example Post"
+date: "2023-12-01"
+---
+
+# Welcome
+
+**Hello world**, this is my first Jekyll blog post.
+
+I hope you like it!
+```
+
+### Setting a Post Thumbnail Image
+
+Add `thumbnail` attribute to the post that you'd like to show a representative image when rendered.
+
+```
+---
+title: "Example Post: thumbnail exists"
+date: "2023-12-02"
+thumbnail: "/assets/img/thumbnail/bricks.webp"
```

Comment:
Did you check if giving an external URL for the thumbnail path also works?

If so, it could be better to let users know that.

Reply:
There is a good example of it.

https://github.com/byanko55/jekyll-theme-satellite/pull/30/files#diff-d25039545e43f2c5319b5ac746c19f93b5563bf9f6bec372a49243c2f50e224a

Reply:
Wouldn't someone find out by himself, maybe?